DataFormats/Detectors/TRD/include/DataFormatsTRD/RawData.h

```diff
@@ -123,18 +123,23 @@ struct TrackletHCHeader {
   union {
     //             10987654321098765432109876543210
     // uint32_t:   00000000000000000000000000000000
-    //                 cccccccccccccccc iiiiiiiiiii
-    //             ffff|               y|
-    //             |   |               |------       0-10 half chamber id
-    //             |   |               ------------- 11 always 0x1
-    //             |   ----------------------------- 12-72 MCM Clock counter
+    //                 cccccccccccccccX LLL   SSSSS
+    //             ffff|              |y|  sss|
+    //             |   |              |||  |  |-----  0-4  supermodule
+    //             |   |              |||  |--------  5-7  stack
+    //             |   |              ||------------  8-10 layer
+    //             |   |              |------------- 11    always 0x1
+    //             |   |              |------------- 12 side of chamber
+    //             |   ----------------------------- 13-72 MCM Clock counter
```

Comment:
We have to make sure that the upper/lower 16 bit cannot be a tracklet end marker (0x1000). This is ok for the lower bits, for the upper bits we are restricted not to use format 8 (I think we mentioned this in the TDP).

Reply:
Sure, but the format is unchanged.